pkg/clients/coredata/event.go

```diff
@@ -469,3 +470,38 @@ func (e *EventRestClient) DeleteOld(age int) error {
 
 	return nil
 }
+
+// Mark event as pushed
+func (e *EventRestClient) MarkPushed(id string) error {
+	req, err := http.NewRequest(http.MethodPut, e.url+"/id/"+id, nil)
+	if err != nil {
+		fmt.Println(err)
```

Comment:
Please do not print on console. Log instead.

Reply:
@drasko looks like all other methods print on console in this source file. Do you want to change them all ?

Reply:
@drasko  And since we always return the errors to the client, do we really need to log the errors ? We probably can let the client decide what to do with the errors. In some cases, the client may not want it logged.

Reply:
@drasko 
Please see the content of this file
https://github.com/edgexfoundry/edgex-go/blob/master/pkg/clients/coredata/event.go

There are lots of places using fmt.Println in this file, so it is another issue.  David just followed the same convention in this file.  We should create another issue and PR to fix this.  Please help merge this PR first.  Thanks.

@SteveOss  please help comment.  Thanks.